src/ethereum_test_tools/common/conversions.py

```diff
@@ -0,0 +1,174 @@
+"""
+Common conversion methods.
+"""
```

Comment:
Really like these in a separate file!! I think at some point we should consolidate on these conversions, and maybe create class types for `Address` & `Hash` to make conversions easier

I added some suggested changes/ideas at the end of `types.py` but I think in the long term we should try to make these as concise as possible or change some of the naming to avoid errors - i.e `hex_or_none` returns a hex string in one case so we could call it `hex_string_or_none` etc

Reply:
Created #182